### PR TITLE
mongodb-replica-set-centos actualized

### PR DESCRIPTION
**Pull request by SychevIgor**
### Changelog
NSG moved to shared-resources.json from primary-resources.json and secondary-resources.json
zabbix rules now optional
added current VM Sizes like d*_v3
fresh version of centos added
deleted parameters without references from primary-resources.json and secondary-resources.json
dropped storage account as it was rudiment from unmanaged disks age.
all vms now inside avset. I don't know how high available installation can be outside of avset.

to test this PR
we need to replace baseUrl in a azuredeploy.json with a version with fixed tempaltes
"baseUrl": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/mongodb-replica-set-centos/",
"baseUrl": "https://raw.githubusercontent.com/SychevIgor/azure-quickstart-templates/mongodb-replica-set-centos-fix/mongodb-replica-set-centos/"

https://github.com/Azure/azure-quickstart-templates/issues/5787

### CODE REVIEW

File: mongodb-replica-set-centos/azuredeploy.json [hunk 1]
```diff
@@ -121,17 +137,10 @@
...
-      "type": "string",
-      "defaultValue": "[resourceGroup().location]",
-      "metadata": {
-        "description": "Location for all resources."
-      }
     }
   },
   "variables": {
-    "baseUrl": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/mongodb-replica-set-centos/",
+    "baseUrl": "https://raw.githubusercontent.com/SychevIgor/azure-quickstart-templates/mongodb-replica-set-centos-fix/mongodb-replica-set-centos/",
```
Comment (bmoore-msft) [inline mongodb-replica-set-centos/azuredeploy.json]: See [BP guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#deployment-artifacts-nested-templates-scripts) for how to handle artifacts

File: mongodb-replica-set-centos/azuredeploy.json [hunk 2]
```diff
@@ -88,7 +92,13 @@
         "Standard_D4",
         "Standard_D2_v2",
         "Standard_D3_v2",
-        "Standard_D4_v2"
+        "Standard_D4_v2",
```
Comment (bmoore-msft) [inline mongodb-replica-set-centos/azuredeploy.json]: let's just remove allowedSizes from these parameters, so we don't have to continue to maintain/update them

**Comment by bmoore-msft** [comment]
Closing abandoned request - please reopen if you can address the feedback.

**Comment by SychevIgor** [comment]
@bmoore-msft only you can reopen it. I don't have such permissions

File: mongodb-replica-set-centos/azuredeploy.json (hunk 1 above)
Comment (SychevIgor) [reply mongodb-replica-set-centos/azuredeploy.json]: @bmoore-msft it was addressed in a second commit a months ago

**Comment by SychevIgor** [comment]
@bmoore-msft

**Comment by bmoore-msft** [comment]
I'm not able to reopen it either - not sure why, but you can just start a new one

**Pull request closed without merging**